src/main/java/org/openmainframeproject/cobolcheck/features/testSuiteParser/TestSuiteParserController.java

```diff
@@ -7,11 +7,15 @@
 import org.openmainframeproject.cobolcheck.services.Messages;
 import org.openmainframeproject.cobolcheck.services.StringHelper;
 import org.openmainframeproject.cobolcheck.services.cobolLogic.NumericFields;
+import org.openmainframeproject.cobolcheck.workers.Generator;
```

Comment:
I am going to ask for some rewriting here. 
TestSuiteParserController is a feature in the structure of COBOL Check. 
As a rule, features should not be importing workers.
You should be able to get the values you are looking for, from a services, instead of a worker.
Or maybe the functionality that you want to implement, should be located in a worker, instead of a feature.
Thank you